lua/damagelogs/shared/autoslay.lua

```diff
@@ -20,7 +27,13 @@ local function CreateCommand()
 					return
 				end
 			end
-			Damagelog:SetSlays(calling_ply, target, rounds, reason, false)
+			Damagelog.markedForSlay[ target ] = {
+				admin = calling_ply,
+				steamid = target,
+				rounds = rounds or 1,
+				reason = reason or "",
```

Comment:
Use here `Damagelog.Autoslay_DefaultReason`

Reply:
Forgot that was an option :P